README.md

```diff
@@ -91,7 +91,7 @@ Please make sure you use Python 2.x. Python 3.x is not supported currently.
 
 #### macOS
 
-To build for macOS, you need [Xcode 11.4](https://developer.apple.com/xcode/downloads/).
+To build for macOS, you need [Xcode 11.4](https://apps.apple.com/us/app/xcode/id497799835?mt=12).
```

Comment:
You could use the Resources page, which has links to all versions (Mac App Store, beta, and older releases).

```suggestion
To build for macOS, you need [Xcode 11.4](https://developer.apple.com/xcode/resources/).
```